### DISCUSSION TITLE
.NET Aspire 8.0.0 GA has been released!

### DISCUSSION THREAD

**Discussion by joperezr** [microsoft/aspire#4258] [Announcements]
.NET Aspire 8.0 GA is now available!

To find out more about the release please visit our [.NET Blog announcement](https://devblogs.microsoft.com/dotnet/dotnet-aspire-general-availability/), and you can also find [.NET Aspire Docs here](https://learn.microsoft.com/dotnet/aspire/) in order to get started with .NET Aspire.

Your experience and satisfaction are our top priorities, and we are eager to hear your thoughts. We value your feedback immensely, as it helps us to continually improve and provide you with the best possible solutions. Please do not hesitate to share any feedback, suggestions, or concerns you may have about .NET Aspire.

Thank you for your support and for being part of our community! 😃

**Comment by wilsonneto-swe** [comment]
Hi @joperezr,

I think it might be a good time to update the top banner/pinned topic. 

This update could help avoid confusion regarding the current version for newcomers, ensuring a smooth adoption.

I'm sorry to bother you with this, and thanks for maintaining this amazing project :)

**Reply by davidfowl** [reply]
Unpinned it

**Reply by wilsonneto-swe** [reply]
Thanks, @davidfowl!
